docs/consume-packages/Central-Package-Management.md

```diff
@@ -126,33 +152,38 @@ You can override an individual package version by using the `VersionOverride` pr
 <Project>
 ```
 
-You can disable this feature by setting the MSBuild property `EnablePackageVersionOverride` to `false` in a project or in a `Directory.Packages.props` or `Directory.Build.props` import file:
+You can disable this feature by setting the MSBuild property `EnablePackageVersionOverride` to `false` in a project or in a `Directory.Packages.props` or
+`Directory.Build.props` import file:
 
 ```xml
-  <PropertyGroup>
-    <EnablePackageVersionOverride>false</EnablePackageVersionOverride>
-  </PropertyGroup>
+<PropertyGroup>
+  <EnablePackageVersionOverride>false</EnablePackageVersionOverride>
+</PropertyGroup>
 ```
 
-When this feature is disabled, specifying a `VersionOverride` on any `<PackageReference />` item will result in an error at restore time indicating that the feature is disabled.
+When this feature is disabled, specifying a `VersionOverride` on any `<PackageReference />` item will result in an error at restore time indicating that
+the feature is disabled.
 
 ## Disabling Central Package Management
 
-If you'd like to disable central package management for any reason, you can disable this feature with the following:
+If you'd like to disable central package management for any a particular project, you can disable it by setting the MSBuild property
+`ManagePackageVersionsCentrally` to `false`:
 
 ```xml
-<ManagePackageVersionsCentrally>false</ManagePackageVersionsCentrally>
+<PropertyGroup>
+  <ManagePackageVersionsCentrally>false</ManagePackageVersionsCentrally>
+</PropertyGroup>
 ```
 
 ## Warning when using multiple package sources
 
-When using central package management, you will see a `NU1507` warning if you have more than one package source defined in your configuration. To resolve this warning, map your package sources with [package source mapping](https://aka.ms/nuget-package-source-mapping) or specify a single package source.
+When using central package management, you will see a `NU1507` warning if you have more than one package source defined in your configuration. To resolve
+this warning, map your package sources with [package source mapping](https://aka.ms/nuget-package-source-mapping) or specify a single package source.
 
 ```
-There are {0} package sources defined in your configuration. When using central package management, please map your package sources with package source mapping (https://aka.ms/nuget-package-source-mapping) or specify a single package source.
+There are 3 package sources defined in your configuration. When using central package management, please map your package sources with package source mapping (https://aka.ms/nuget-package-source-mapping) or specify a single package source.
 ```
 
-> [!Note]
-> This feature is in active development. We appreciate you trying it out and providing any feedback you may have at [NuGet/Home](https://github.com/nuget/home/issues).
->
-> * There is currently no support in Visual Studio or the .NET CLI for Central Package Management.
+## Additional Notes
```

Comment:
Sure we don't want to keep the existing callout? Good however you see fit.

Reply:
Sorry I was looking at how it rendered in markdown and didn't realize this syntax actually works when rendered as a document.  I'll revert that part of it.